docs/topics/auth/default.txt

```diff
@@ -1831,7 +1838,12 @@ Creating users
 You should see a link to "Users" in the "Auth"
 section of the main admin index page. The "Add user" admin page is different
 than standard admin pages in that it requires you to choose a username and
-password before allowing you to edit the rest of the user's fields.
+password (or setting the password as unusable) before allowing you to edit the rest of
```

Comment:
I forgot to mention that docs should always be wrapped at 79cols, so this line may need an adjustment.